md files/user_stories.md

```diff
@@ -0,0 +1,24 @@
+This is an app used in a focus group to collect data on user preferences of potential products they would purchase.
```

Comment:
I would put all the `.md` files in the root of the project for easy access. (And GitHub's special handling of `README.md`.)

The actual items on this list look good, but please update them in the "As a <role> I want <feature> so that <value>." structure found in the user stories for past assignments. If you're not sure how to structure them, take a look at your peers' user stories documents.